docs/source/installation.md

```diff
@@ -11,6 +11,11 @@ conda activate pymc_env
 ```
 If you like, replace the name `pymc_env` with whatever environment name you prefer.
 
+:::{seealso}
+The [conda-forge tips & tricks](https://conda-forge.org/docs/user/tipsandtricks.html#using-multiple-channels) page to avoid installation
+issues when using multiple conda channels (e.g. defaults and conda-forge)  
+:::
+
```

Comment:
```suggestion
:::{seealso}
The [conda-forge tips & tricks](https://conda-forge.org/docs/user/tipsandtricks.html#using-multiple-channels) page to avoid installation
issues when using multiple conda channels (e.g. defaults and conda-forge)
:::
```